src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -80,6 +80,17 @@ public static TestCommand FromArgs(string[] args, string msbuildPath = null)
                 msbuildPath);
         }
 
+        private static string DefaultVerbosity(AppliedOption options)
```

Comment:
Nit: Missing a verb in the method name, something like `GetDefaultVerbosity`.

Reply:
renamed